src/test-helpers.js

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import { cloneDeep } from 'lodash'
+import { cloneDeep } from 'es-toolkit'
```

Comment:
This one is JSON payload

Reply:
Yes, but it is only used in tests.

Some other places in tests require `cloneDeep` anyway.

Alternative here is `structuredClone`, but then it requires `toRaw` in Vue 3 for reactive objects.